.github/workflows/build.yml

```diff
@@ -597,11 +598,13 @@ jobs:
         run: yarn test:ci${{ matrix.project && format(' --project={0}', matrix.project) || '' }}${{ matrix.shard && format(' --shard={0}/{1}', matrix.shard, matrix.shards) || '' }}
 
       - name: Upload Playwright Traces
-        uses: actions/upload-artifact@v3
-        if: always()
+        uses: actions/upload-artifact@v4
+        if: failure()
```

Comment:
I adjusted this to only store these on failure, I think this is good enough? Then we do not have to store all the traces all the time when we do not need them.

Reply:
Traces should in general only be generated on failure - making this kinda moot - but we will save the ci time for the job running I guess.

Reply:
Jup, it seems to work, see here: https://github.com/getsentry/sentry-javascript/actions/runs/10593166442/attempts/1 there are two artifacts, for the two failed jobs.